README.md

```diff
@@ -10,6 +10,17 @@ Wish list - web app, where users can:
 
 Project requirements:
 - Maven 3+
-- Postgres 9+
 - Java 8
+- Docker 18
```

Comment:
It is hard to install, why do we need it? Project doesn't have many system dependencies. So docker doesn't solve any problem. But it is much harder to maintain than PostgreSQL.

Reply:
I don't want to install Postgres for running this project. I don't want to install any other application on which the project may depend.

Reply:
So main contibutor don't want to install docker.